content/blog/2023/2023-09-25-introduction-to-kotest.md

```diff
@@ -0,0 +1,239 @@
+---
+title: "Introduction to Kotest"
+categories: ["Kotlin"]
+date: 2023-09-25 00:00:00 +1100 
+authors: [ezra]
+excerpt: "In this tutorial, we'll discuss Kotest which is a testing Framework  written in Kotlin."
+image: images/stock/0104-on-off-1200x628-branded.jpg
+url: introduction-to-Kotest
+---
+
+
+
+Kotest is simply a multi-platiform framework used for testing and it's written in Kotlin.In this tutorial, we shall cover the following sub-topics related to the Kotest framework: testing with Kotest, testing styles used, grouping Kotest tests with tags, the lifecycle hooks and assertions supported by Kotest.
+
+## Testing on JVM
```

Comment:
What's the alternative to "testing on JVM"? There must be an alternative if you use this header for this section.